stars/game.py

```diff
@@ -108,7 +113,10 @@ def __init__(self, x=50, y=50, z=50, num_systems=10, system_names=None, races=[]
                 for p in self.players:
                     p.add_intel(b, {'location': b.location, 'size': b.radius})
             self._scan([])
-            self._call(self.players, 'update_stats')
+            if self.hundreth == 0:
+                self.calculate_score()
+            else:
```

Comment:
It's bad coding practice to assume that the calculate_score will call update_stats.  Better to just always call update_stats before calling calculate_score

Reply:
calculate score calls update status within itself